app/assets/javascripts/login_activity/LoginActivityPage.test.js

```diff
@@ -1,18 +1,41 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import LoginActivityPage from './LoginActivityPage';
 import fetchMock from 'fetch-mock/es5/client';
+import LoginActivityPage from './LoginActivityPage';
+import loginActivityJson from './loginActivityJson.fixture';
 
-it('renders without crashing', () => {
+function mockFetch(props = {}) {
   fetchMock.restore();
-  fetchMock.get(
-    '/api/login_activity?created_at_or_before=1532105716&created_after=1529513716',
-    '[]'
-  );
 
+  const {queryStartTimestamp, queryEndTimestamp} = props;
+  const url = `/api/login_activity?created_at_or_before=${queryEndTimestamp}&created_after=${queryStartTimestamp}`;
+  fetchMock.get(url, loginActivityJson);
+}
+
+function testProps(props = {}) {
+  return {
+    queryStartTimestamp: '1529513716',
+    queryEndTimestamp: '1532105716',
+    ...props
+  };
+}
+
+it('renders without crashing', () => {
+  const props = testProps();
+  mockFetch(props);
   const el = document.createElement('div');
-  ReactDOM.render(
-    <LoginActivityPage
-      queryStartTimestamp={'1529513716'}
-      queryEndTimestamp={'1532105716'} />, el);
+  ReactDOM.render(<LoginActivityPage {...props} />, el);
+});
+
+it('renders after fetching', done => {
```

Comment:
Since the component fetches data, that promise doesn't resolve synchronously and so by the time the first test finishes, none of the rendering code has run.  This second test here is an async test that uses `done`, where we do the same thing, but then wait for the promise resolve and the rendering to happen before making assertions and telling Jest the test is done.

The setTimeout value is`0` milliseconds, which tells the JS engine to let any queued up work on the event loop go first (ie, for the mock fetch that we setup to immediately resolve to the fixture, run any work that's waiting on that).  So by the time the function in setTimeout is run, the rendering has already happened and we can make assertions, etc. and then tell Jest the test is done.